templates/workloads/services/lb-web/cf.yml

```diff
@@ -168,6 +167,31 @@ Resources:
               Action:
                 - elasticloadbalancing:DescribeRules
               Resource: "*"
+        - PolicyName: "EnvControllerStackUpdate"
```

Comment:
Hmmm we are increasing the permission scope here! Which would require a security review.

I'm trying to understand how come this prototype worked: https://github.com/efekarakus/copilot-cli/commit/66a2c275c54cfa58b34e90140a84b6c4121ce6c1 without these permission changes. I feel like my prototype shouldn't have worked :O strange!

Reply:
How do you perform those actions without permission 🤔

Reply:
ok instead of increasing the scope of the other Lambda, let's create a new role:
`EnvControllerLambdaRole` which has the permissions that you've listed.

Can we try being specific on the resources? For example:
1. the stack
```yaml
Resource:  !Sub 'arn:aws:cloudformation:${AWS::Region}:${AWS::AccountId}:stack/${AppName}-${EnvName}/*'
```
2. the role
```yaml
Resource:  !Sub 'arn:aws:iam::${AWS::AccountId}:role/${AppName}-${EnvName}-CFNExecutionRole'
```

Reply:
Agree on creating a new role otherwise the other lambda scope might be expanded as well. However, I don't get why we want to be specific on the resources? I feel they are about the same since the only stack with both copilot app and env tags are in format of those two. But one benefit for using tags is it is agnostic about the arn format, whereas we are very unlikely to change our copilot tags. What do you think?

Reply:
I guess we also need to move the role for auto scaling out right? i'll do that in the next PR

Reply:
The slight difference is that if we use only tags, then the role can also update other service stacks in the environment. Whereas specifying the ARN limits it strictly to the environment stack and not to the services.

Reply:
ah got it good point!